app/views/admin/document_collection_groups/index.html.erb

```diff
@@ -79,10 +79,10 @@
                 <li class="remove">
                   <%= submit_tag 'Remove', class: 'btn btn-sm btn-default' %>
                 </li>
-                <% if other_groups(group).size > 0 %>
+                <% if (@groups - [group]).size > 0 %>
```

Comment:
I'd be tempted to precede this with a `other_groups = @groups - [group]` to make the code slightly more self documenting and dry. Does the job of killing off a redundant module while keeping the benefits it was trying to encapsulate.

Reply:
Sure, I've added a view variable here.